### ISSUE TITLE
v0.9.1 package missing lib/index.js

### ISSUE THREAD

**Issue by liffiton** [jupyterhub/jupyterlab-hub#60]
The just-uploaded v0.9.1 fails to install:

```
$ npm pack @jupyterlab/hub-extension

npm notice
npm notice package: @jupyterlab/hub-extension@0.9.1
npm notice === Tarball Contents ===
npm notice 907B  package.json
npm notice 2.0kB README.md
npm notice 1.5kB LICENSE
npm notice === Tarball Details ===
npm notice name:          @jupyterlab/hub-extension
npm notice version:       0.9.1
npm notice filename:      jupyterlab-hub-extension-0.9.1.tgz
npm notice package size:  2.2 kB
npm notice unpacked size: 4.4 kB
npm notice shasum:        b00eb67ed24454dacf802ad33bdf529becc9d5d8
npm notice integrity:     sha512-QZJtYhtNietbq[...]jEvHqzlr5DdQA==
npm notice total files:   3
npm notice
jupyterlab-hub-extension-0.9.1.tgz

Errored, use --debug for full output:
ValueError: "@jupyterlab/hub-extension" is not a valid extension:
Missing extension module "lib/index.js"
```

**Comment by gveltri** [comment]
Similarly having an issue installing with 

```
jupyter labextension install @jupyterlab/hub-extension
npm pack @jupyterlab/hub-extension
```

and getting the same error:

```
ValueError: "@jupyterlab/hub-extension" is not a valid extension:
Missing extension module "lib/index.js"
```

**Comment by gveltri** [comment]
Thanks @dfsnow, I was wondering how to version lock

**Comment by dhirschfeld** [comment]
ping!

My automated build just failed because of this. Thanks for the workaround @liffiton - I'll add that to my Dockerfile...

**Comment by ian-r-rose** [comment]
Hi all, thanks for flagging this! There was indeed an error in the publishing of v0.9.1. I just published an v0.9.2 to fix it, let me know if that works for you.

**Comment by liffiton** [comment]
Works for me.  Thanks!
